packages/react-day-picker/src/contexts/DayPicker/DayPickerContext.tsx

```diff
@@ -48,26 +48,28 @@ export interface DayPickerContextValue extends DayPickerProps {
 }
 
 /**
- * The DayPicker Context shares the props passed to DayPicker within the
- * internal components. It is used to set the default values and perform
+ * The DayPicker Context shares the props passed to DayPicker within internal
+ * and custom components. It is used to set the default values and perform
  * one-time calculations required to render the days.
  *
- * Access this context from the [[useDayPicker]] hook when using custom
- * components.
+ * Developers may access to this context from the [[useDayPicker]] hook when
```

Comment:
Either: “developers may get access to this” or “developers may access this”?

Reply:
Better! THanks1